apps/journeys-admin/src/components/Editor/Toolbar/Items/StrategyItem/StrategyItem.spec.tsx

```diff
@@ -2,22 +2,57 @@ import { MockedProvider } from '@apollo/client/testing'
 import { fireEvent, render, waitFor } from '@testing-library/react'
 import { SnackbarProvider } from 'notistack'
 
-import { EditorProvider } from '@core/journeys/ui/EditorProvider'
+import {
+  ActiveContent,
+  ActiveFab,
+  EditorState,
+  useEditor
+} from '@core/journeys/ui/EditorProvider'
+import {
+  ActiveCanvasDetailsDrawer,
+  ActiveSlide,
+  EditorProvider
+} from '@core/journeys/ui/EditorProvider/EditorProvider'
 import { JourneyProvider } from '@core/journeys/ui/JourneyProvider'
 
 import { JourneyFields } from '../../../../../../__generated__/JourneyFields'
 
 import { StrategyItem } from './StrategyItem'
 
+import '../../../../../../test/i18n'
+
+jest.mock('@core/journeys/ui/EditorProvider', () => {
+  const originalModule = jest.requireActual('@core/journeys/ui/EditorProvider')
+  return {
+    __esModule: true,
+    ...originalModule,
+    useEditor: jest.fn()
+  }
+})
+
 describe('StrategyItem', () => {
   const mockJourney: JourneyFields = {
     id: 'journeyId',
     title: 'Some Title',
     slug: 'journeySlug'
   } as unknown as JourneyFields
 
+  const mockUseEditor = useEditor as jest.MockedFunction<typeof useEditor>
+
+  const dispatch = jest.fn()
+  const state: EditorState = {
+    activeFab: ActiveFab.Add,
+    activeSlide: ActiveSlide.Content,
+    activeContent: ActiveContent.Canvas,
+    activeCanvasDetailsDrawer: ActiveCanvasDetailsDrawer.Properties
+  }
+
   beforeEach(() => {
     jest.clearAllMocks()
+    mockUseEditor.mockReturnValue({
+      state,
+      dispatch
+    })
   })
 
   it('should call close menu', async () => {
```

Comment:
Close menu is really only one part of what this test is doing, so we should rename it to be descriptive of the entire test
```suggestion
  it('should goals click', () => {
```